tests/data/packages/TopoRequires4/setup.cfg

```diff
@@ -0,0 +1,6 @@
+[metadata]
```

Comment:
Its not, OTOH I'm lazy and don't want to regenerate the two tarballs containing it :). But, I can remove if needed.